INSTALL.rst.txt

```diff
@@ -14,13 +14,13 @@ Prerequisites
 
 Building NumPy requires the following installed software:
 
-1) Python__ 3.7.x or newer.
+1) Python__ 3.8.x or newer.
 
    Please note that the Python development headers also need to be installed,
    e.g., on Debian/Ubuntu one needs to install both `python3` and
    `python3-dev`. On Windows and macOS this is normally not an issue.
 
-2) Cython >= 0.29.28
+2) Cython >= 0.29.29 but < 3.0
 
 3) pytest__ (optional) 1.15 or later
```

Comment:
As far as I see this requirement is not stated in any config files, maybe consider adding it as `minversion` to pytest.ini?

Reply:
This 1.15 is not a pytest version, is it supposed to be numpy's? Either case, I would suggest to add a `minversion`  to the config and document it here.

Reply:
I don't think we have a hard dependency on any pytest version features, so maybe just remove the `(optional) 1.15 or later`

Reply:
Cython 0.29.30 was also released as a bugfix for 0.29.29.

Reply:
> so maybe just remove the `(optional) 1.15 or later`

Case in point: pytest 1.15 [does not](https://github.com/pytest-dev/pytest/tags?after=2.0.1) exist (and even if it did, it would be _ancient_)